lib/system.js

```diff
@@ -10,6 +10,7 @@ const rsync = require('./rsync'),
       util = require('./util');
 
 const sequelize = require('sequelize');
+const { importantLogs } = require('./util');
```

Comment:
You don't need to import this here because `util` is already imported above (line 10). Instead, you can just call `util.importantLogs(...)`